04_environment/index.tex

```diff
@@ -1 +1,283 @@
+    \label{fig:Pareto Optimal Strategy}
+\end{figure}
+
+\subsubsection{Social Welfare Maximisation}
+
+It is possible for these islands to find the social welfare maximum in this foraging function with enough time. The islands could identify the exact amount of resources needed, for both fishing and deer hunting, to achieve the best return according to the maximum number of animals. Thus by cooperating who goes where and spends how much they are able to maximize their returns without spending more resources than necessary.
+
+\subsection{Extended Work}
+
+For the future implementations, our additions focus on further exploring the dynamics of our agents and allowing for a more complex decision-making interaction between clients and the environment. Some of these design ideas were later implemented in self-organising multi-agent system and others are expressed for the sake of completeness and future work recommendations.
+
+Some changes have been made to make foraging more volatile and unpredictable. These will add to the complexity of the system, allowing the teams to further evaluate the agents’ performance on observing and using learned knowledge to take action.
+
+\subsubsection{Incorporated Changes: Population Density}
+
+Introducing a model that controls the population density of deer allows us to increase the complexity of the foraging function. In short, the deer hunting capacity decreases with decreasing population and increases with an increasing population. This will then influence the maximum deer per hunt parameter(n), which in turn results in a more complex return calculation function (\texttt{\textbf{DtotalReturn}}), which would make it harder for agents to figure out forage tier boundaries to optimise their strategies.
+
+The population change can be dynamic depending on various environmental factors, fetched from the rest of the system. For example, a disaster could supposedly cause the population density of deer to halve, making it harder to forage and therefore, causing lower returns. For future implementations, the team uses a single species population model and more specifically logistic modelling for the deer and fish population. The model can be described in Equation~\eqref{eq:Population Density}.
+
+\begin{equation}
+\frac{\mathrm{d} P}{\mathrm{~d} t}=k(N-P)
+\label{eq:Population Density}
+\end{equation}
+
+\begin{itemize}
+        \item “$P$” is the total Population. \item “$N$” is the maximum deer/fish population (“carrying capacity” formally).
+        \item “$k$” is the growth coefficient.
+        \item “$t$” is time.
+    \end{itemize}
+
+In our implementation $k$ and $N$ are constants but it would be interesting to see what would happen if the growth coefficient increases, causing a population growth and observing the islands’ dynamics. Will they keep on “investing” higher resources each turn, taking advantage of the deer/fish abundance, or will they stick to their original, safe and static strategy?
```

Comment:
In our implementation,

Reply:
but it would be interesting --> However, it would be

break the long sentence. and start a new one with ''however''